src/operations/threshold.ts

```diff
@@ -87,38 +87,39 @@ export function computeThreshold(
     );
   }
   const histogram = image.histogram({ slots });
+  const scale = options.slots ? 2 ** image.bitDepth / options.slots : 1;
```

Comment:
@targos We didn't have a rule that forbid us to use options.slots when it has been destructured ? Or it is because we send it to image.histogram and eslint does not now it can not be changed by inside the function ?

@moonayyur I think you can use slots and not options.slots.